pandas/core/dtypes/missing.py

```diff
@@ -200,6 +217,11 @@ def notna(obj):
     """Replacement for numpy.isfinite / -numpy.isnan which is suitable for use
```

Comment:
Can you shorten this to a single line? Can be similar to you docstring for `isna`, just flipped.